clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -482,13 +491,18 @@ func (w *waitForControlledPodsRunningMeasurement) waitForRuntimeObject(obj runti
 	o.lock.Lock()
 	defer o.lock.Unlock()
 	w.handlingGroup.Start(func() {
+		var minDesiredPodCount int
+		if w.countErrorMargin > 0 {
+			minDesiredPodCount = int(runtimeObjectReplicas) - w.countErrorMargin
```

Comment:
I would do that slightly different, i.e. always set
minDesiredPodCount = runtimeObjectReplicas - w.countErrorMargin

[i.e always treat it as min] - I think the code would be a bit simpler below]